debian/control

```diff
@@ -10,6 +10,6 @@ Vcs-Git: https://github.com/WhitewaterFoundry/pengwin-setup.git
 
 Package: pengwin-setup
 Architecture: all
-Depends: ${misc:Depends}, git, wslu, pengwin-base (>= 0.1-175)
+Depends: ${misc:Depends}, git, p7zip-full, wslu, pengwin-base (>= 0.1-175)
```

Comment:
It is better that you install p7zip-full with the program which needs it as we do with the other installers in pengwin-setup

Reply:
Okay, will do it separately